### PR TITLE
Add cardview and use glide to load images

### PR DESCRIPTION
**Pull request by hernanhorquin96**
In this PR, i add cardiew element into item character layout and use glide instead of picasso for loading images.
In next PR, i will implement realm database into the project.

marvel

### CODE REVIEW

**Review by BQuinteros — APPROVED** [review]

File: app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/extension/ImageExtensions.kt [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package com.puzzlebench.clean_marvel_kotlin.presentation.extension
 
 import android.widget.ImageView
-import com.squareup.picasso.Picasso
+import com.bumptech.glide.Glide
 
 fun ImageView.getImageByUrl(url: String) {
```
Comment (SartiMau) [inline app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/extension/ImageExtensions.kt:6]: 👏

**Review by SartiMau — APPROVED** [review]

File: app/src/main/res/layout/character_cards_layout.xml [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+        <ImageView
+            android:id="@+id/image_thumbnail"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/super_hero_row_gradient_height"
+            android:background="@drawable/super_hero_gradient"
+            app:layout_constraintBottom_toBottomOf="parent" />
```
Comment (mauroLonghi) [inline app/src/main/res/layout/character_cards_layout.xml:23]: What is the mining of this view?

Comment (lucianoeli) [reply app/src/main/res/layout/character_cards_layout.xml:23]: its the gradient shadow of the card

**Review by lucianoeli — APPROVED** [review]

File: app/src/main/res/layout/character_cards_layout.xml (hunk 2 above)
Comment (hernanhorquin96) [reply app/src/main/res/layout/character_cards_layout.xml:23]: what lucho said is correct!

**Pull request merged**